app/helpers/pipeline_runs_helper.rb

```diff
@@ -181,4 +181,25 @@ def pipeline_version_has_coverage_viz(pipeline_version)
   def pipeline_version_at_least_2(pipeline_version)
     pipeline_version_at_least(pipeline_version, PIPELINE_VERSION_2)
   end
+
+  def check_for_user_error(failed_stage)
+    if failed_stage.step_number == 1
```

Comment:
ok done. Although I was thinking in the future it would probably be more like
```
if failed_stage.step_number == 1
  ...
elsif failed_stage.step_number == 2
  ...
```
etc